packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect, useState, useMemo } from '@wordpress/element';
+import {
+	__experimentalSpacer as Spacer,
+	__experimentalInputControl as InputControl,
+	__experimentalText as Text,
+	SelectControl,
+	Spinner,
+	Icon,
+	FlexItem,
+	Flex,
+	Button,
+} from '@wordpress/components';
+import { debounce } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { search, closeSmall } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import TabLayout from './tab-layout';
+import { FontLibraryContext } from './context';
+import FontsGrid from './fonts-grid';
+import FontCard from './font-card';
+import filterFonts from './utils/filter-fonts';
+import CollectionFontDetails from './collection-font-details';
+import { toggleFont } from './utils/toggleFont';
+import { getFontsOutline } from './utils/fonts-outline';
+import GoogleFontsConfirmDialog from './google-fonts-confirm-dialog';
+
+const DEFAULT_CATEGORY = {
+	id: 'all',
+	name: __( 'All' ),
+};
+function FontCollection( { id } ) {
+	const requiresPermission = id === 'google-fonts';
+
+	const getGoogleFontsPermissionFromStorage = () => {
+		return (
+			localStorage.getItem(
```

Comment:
In other places, the local storage is accessed through the `window` global. This way, it won't need to be listed as a global in the ESLint config.

Reply:
done

Reply:
thank you!